.github/actions/javascript/postTestBuildComment/postTestBuildComment.js

```diff
@@ -0,0 +1,71 @@
+const core = require('@actions/core');
+const {context} = require('@actions/github');
+const GithubUtils = require('../../../libs/GithubUtils');
+
+/**
+ * @returns {String}
+ */
+function getTestBuildMessage() {
+    console.log('Input for android', core.getInput('ANDROID', {required: true}));
```

Comment:
(Not a blocker) - I'm curious why we only log android here and not the other platforms